lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -299,7 +299,59 @@
         description:
           - Whether the mount should be read-only.
         type: bool
-        default: no
+      labels:
+        description:
+          - Volume labels to apply.
+        type: dict
+        version_added: "2.8"
+      propagation:
+        description:
+          - The propagation mode to use.
+          - Can only be used when I(mode) is C(bind).
+        type: str
+        choices:
+          - shared
+          - slave
+          - private
+          - rshared
+          - rslave
+          - rprivate
+        version_added: "2.8"
+      no_copy:
+        description:
+          - Disable copying of data from a container when a volume is created.
+          - Can only be used when I(mode) is C(volume).
+        type: bool
+        version_added: "2.8"
+      driver_config:
+        description:
+          - Volume driver configuration.
+          - Can only be used when I(mode) is C(volume).
+        suboptions:
+          name:
+            description:
+              - Name of the volume-driver plugin to use for the volume.
+            type: str
+          options:
+            description:
+              - Options as key-value pairs to pass to the driver for this volume.
+            type: dict
+        type: dict
+        version_added: "2.8"
+    tmpfs_size:
```

Comment:
This and `tmpfs_mode` should be indented two more spaces.

Reply:
Yup, The indentation error actually caused the sanity check failure. Had to run the sanity check in python 3 to get the error locally.